survey-runner-application/run-terraform.sh

```diff
@@ -35,7 +35,7 @@ if [ -z "$public_subnet_ids" ]; then
 fi
 
 # get the private route table ids as a comma separated list
-private_route_table_ids=`aws ec2 describe-route-tables --filters "Name=tag:Environment,Values=${AWS_ENVIRONMENT_NAME}" "Name=tag:Type,Values=Private" --query 'RouteTables[*].RouteTableId' --output text | tr '\t' ','`
+private_route_table_ids="[$(aws ec2 describe-route-tables --filters "Name=tag:Environment,Values=${AWS_ENVIRONMENT_NAME}" "Name=tag:Type,Values=Private" --query 'RouteTables[*].RouteTableId' --output text | tr '\t' ',' | sed -e 's/,/","/g')\"]"
```

Comment:
I get an error when running this:
```
invalid value "private_route_table_ids=[rtb-cc1d56ab\",\"rtb-ed13588a\",\"rtb-091c576e\"]"
```
Looks like a missing `"` before the first route table id